app/src/main/java/com/sdp/movemeet/backend/firebase/firestore/FirestoreManager.java

```diff
@@ -32,14 +32,14 @@ public FirestoreManager(FirebaseFirestore db, String collection, BackendSerializ
         object.setDocumentPath(documentPath);
         Map<String, Object> data = serializer.serialize(object);
 
-        return db.collection(collection).document(object.getDocumentPath()).set(data);
+        return db.document(object.getDocumentPath()).set(data);
```

Comment:
Will the query still be able to find the exact document if we don't specify the collection (e.g "Activities") ?

Reply:
this change comes from Kepler who explained to us at a meeting that indeed, there would be no problems. I prefer not to explain too much for fear of being wrong and I advise you to ask him directly to be sure and have the exact reason ;)